stylesheets/_component.calendar.scss

```diff
@@ -5,7 +5,6 @@
 }
 
 .ui-datepicker {
-    padding-bottom: 1em;
```

Comment:
Is this relevant to this change?

Reply:
Yes and no. It causes additional padding at the bottom of every page (like the footer) which also causes additional scroll on the right-hand content as, unlike the footer, it isn't counteracted in sticky mode.

It looks like this CSS rule is overridden when the date picker shows by in-line styles. So it's essentially useless.

<img width="685" alt="Screenshot 2020-09-22 at 17 37 48" src="https://user-images.githubusercontent.com/9383439/93911574-7b5df480-fcfa-11ea-88aa-e3d6daed73e4.png">